packages/langium/src/index.ts

```diff
@@ -27,6 +27,7 @@ export * from './serializer/json-serializer';
 export * from './service-registry';
 export * from './services';
 export * from './syntax-tree';
+export * from './typification/type-collector';
```

Comment:
The type collector is related to the grammar language and is not meant to be used in custom DSLs. Please move it to the `grammar` subfolder.